src/cli/commands/__tests__/lint.test.ts

```diff
@@ -96,6 +96,20 @@ describe('lint', () => {
         });
       });
     });
+
+    test
+      .stdout()
+      .command(['lint', 'src/__tests__/__fixtures__/gh-474/spec.yaml'])
+      .it('should handle relative path to a document', ctx => {
```

Comment:
Sorry I didn’t spit before but can we assert a positive as proof? This sort of double negative doesn’t catch against situations where instead of invalid-ref we change the code to say broken-ref and nobody knows that refs aren’t working properly anymore. We can keep the negative assertion but we should add positive assertions too.

Reply:
> We can keep the negative assertion but we should add positive assertions too.

Yeah, I was thinking of it as well.
I'll introduce some errors in a referenced document.

Reply:
Ha, adding a test made me discover a subtle and unrelated to the original issue error.

Reply:
Alright, the above is related to caching... which made me discover yet another bug :laughing: this time in json-ref-resolver